### ISSUE TITLE
Unable to configure Ambassador using instructions

### ISSUE THREAD

**Issue by kimschles** [knative/docs#4646]
## Expected Behavior
In the 'Installing with the Operator' documentation, I expect to follow the instructions labeled 'The following commands install Ambassador and enable its Knative integration' under the Ambassador Tab and be able to successfully complete all 7 steps. 

## Actual Behavior
In [step 4](https://github.com/knative/docs/blob/main/docs/install/operator/knative-with-operators.md?plain=1#L330-L334), when I run the command 

```console
kubectl set env --namespace ambassador  deployments/ambassador AMBASSADOR_KNATIVE_SUPPORT=true
``` 

I get the error 
```console 
Error from server (NotFound): deployments.apps "ambassador" not found
```

## Steps to Reproduce the Problem

After installed the Knative Operator and Knative Serving Component, work through the Amabssador instructions starting [here](https://github.com/knative/docs/blob/main/docs/install/operator/knative-with-operators.md?plain=1#L307).

## Additional Info
**Additional context**
n/a

**Install information:**
 - Platform (GKE, IKS, AKS, etc.): DigitalOcean Managed Kubernetes 
 - Knative Version: `v1.1.0`

**Comment by snneji** [comment]
Hi @kimschles, thanks for submitting an issue!

@shinigambit, since you're on interrupt for Knative.party this week, could you help out with this one?

**Comment by shinigambit** [comment]
The issue appears to be that the Deployment definition already specifies the "default" namespace in https://getambassador.io/yaml/ambassador/ambassador-rbac.yaml and kubectl doesn't honor the `--namespace ambassador` flag as used in the second step, I'll check what options we have to fix this

**Comment by shinigambit** [comment]
@snneji , @kimschles it turns out that Knative has dropped support for Ambassador for a while now, the docs just were outdated, instructions for installing Ambassador are being removed by this pr #4670

**Comment by snneji** [comment]
Closing PR as it has been resolved in #4670
